packages/reports/src/Constants/data-table-constants.tsx

```diff
@@ -25,20 +37,43 @@ const getModeFromValue = key => {
         transfer: 'transfer',
     };
 
-    if (Object.keys(map).find(x => x === key)) {
-        return map[key];
-    }
+    return map[key] || map.default;
+};
+
+type TCellContentProps = {
+    cell_value: string;
+    passthrough: any;
```

Comment:
Can we use another type instead of `any`? maybe we can use generic types if you are not sure about the type

Reply:
I think we better use `any` type for `passthrough` object, this object contains different values depending on from which component it's passed, and we might want to use it for different purposes in the following components. If we specify any strict datatype, it will be troublesome to pass new values and will cause a lot of inconvenience for developers. The same concerns I have about row_obj, since it varies differently among components and should be extendable.